docs/themes/modifying_themes.md

```diff
@@ -41,4 +41,91 @@ The result in the default theme will look like::
+
+-   the header,
+-   the footer,
+-   the call for the page content (`<?php echo $this->content; ?>`),
+-   calls to fonts, css, and javascript files.
+
+Themes load their CSS and Javascript files using Zend's view helper functions ([`headLink()`](https://docs.zendframework.com/zend-view/helpers/head-link/) and [`headScript()`](https://docs.zendframework.com/zend-view/helpers/head-script/), which help control the order in which these files load alongside files from Omeka S's pages and  modules. These functions should appear before the `<head>` of layout.phtml.
+
+Here are examples for including these files in the <head> of a theme.
+
+For CSS:
+
+`$this->headLink()->prependStylesheet($this->assetUrl('css/style.css'));`
+
+The function `prependStylesheet()` will place the stylesheet at the top of the queue of loaded css files.
+
+`$this->headLink()->appendStylesheet($this->assetUrl('css/style.css'));`
+
+The function `appendStylesheet()` will place the stylesheet at the bottom of the queue of loaded CSS files.
+
+For Javascript:
+
+`$this->headScript()->prependFile($this->assetUrl('js/thedaily.js'));`
+
+The function `prependFile()` will place the stylesheet at the top of the queue of loaded javascript files.
+
+`$this->headScript()->appendFile($this->assetUrl('js/thedaily.js'));`
+
+The function `appendFile()` will place the javascript file at the top of the queue of loaded javascript files.
+
+### Using Webfonts
+
+Themers have access to a greater variety of fonts to use via hosted webfont libraries. Common examples include free resources like [Google Fonts](https://fonts.google.com) and [Adobe Edge Web Fonts](https://edgewebfonts.adobe.com/), as well as paid subscription sites such as [Fonts by Hoefler & Co](https://www.typography.com/webfonts/) and [Mosaic](https://www.monotype.com/fonts/mosaic). Hosted webfont usage often looks like the following:
+
+`<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">`
+
+This line sources the font family “Lato” hosted by Google Fonts. To start using a library's webfont, look for where they provide a similar markup line. This line essentially calls a stylesheet that imports the hosted font files to a page. As such, themers will use the functionality to include a CSS file for the purposes of including a webfont in layout.phtml. For example:
+
+`<?php $this->headLink()->prependStylesheet('//fonts.googleapis.com/css?family=Lato); ?>`
+
+## Working with Omeka S Content
+
+Omeka S's sites are primarily concerned with the presentation of Omeka S resources: items, item sets, and media. Themes control how these resources' associated metadata and files are displayed via methods defined in the resources' representations. The following section highlights some common tasks using these methods, and a more in-depth guide can be found in [Using Representations](https://omeka.org/s/docs/developer/key_concepts/representations/#using-representations).
+
+#### Displaying Resource Images
```

Comment:
We should probably talk about the `thumbnail` helper here actually...

```
$this->thumbnail($someResource, 'thumbnailSize', [/* array of attributes */]);
```

It does primaryMedia for you and won't give you anything but an image. If your media is a video or audio or whatever, `render` will get you the player. Plus it's what the custom thumbnail support runs through.